subprojects/plugins/src/integTest/groovy/org/gradle/api/plugins/BasePluginIntegrationTest.groovy

```diff
@@ -37,7 +39,8 @@ class BasePluginIntegrationTest extends AbstractIntegrationSpec {
         fails "clean"
 
         then:
-        failure.assertHasCause("Unable to delete file")
+        failure.assertHasCause("Unable to delete directory '${file('build')}'")
+        failure.assertThatCause(containsString(file("build/newFile").absolutePath))
```

Comment:
We would expect this to fail with the "failed to delete children" message - I would think we should check for that here, too.

Reply:
It's not exactly like that. All files existing at deletion time were deleted, none failed, but new files were found after the failure. Makes sense?